src/App.vue

```diff
@@ -51,25 +61,25 @@ export default {
   name: "App",
   data() {
     return {
-      isIframe: (window.self !== window.top),
+      isIframe: window.self !== window.top,
       loading: true,
       loadingProgress: 0,
-      loadingPollInProgress: false
+      loadingPollInProgress: false,
     };
   },
   computed: {
     ...mapState({
-      hasShutdown: state => state.system.hasShutdown,
-      shuttingDown: state => state.system.shuttingDown,
-      rebooting: state => state.system.rebooting,
-      isManagerApiOperational: state => state.system.managerApi.operational,
-      isApiOperational: state => state.system.api.operational,
-      jwt: state => state.user.jwt,
-      updateStatus: state => state.system.updateStatus
+      hasShutdown: (state) => state.system.hasShutdown,
+      shuttingDown: (state) => state.system.shuttingDown,
+      rebooting: (state) => state.system.rebooting,
+      isManagerApiOperational: (state) => state.system.managerApi.operational,
+      isApiOperational: (state) => state.system.api.operational,
+      jwt: (state) => state.user.jwt,
+      updateStatus: (state) => state.system.updateStatus,
```

Comment:
Do you have something configured in your editor to auto add all these formatting changes? It makes it a little hard to see what's changed in the diff. (we really need to setup project wide formatting to resolve this as a priority)